test/test-proxied-properties.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const {assert} = require("chai");
+const sinon = require("sinon");
+
+const {setupTestDOMWindow} = require("./setup");
+
+describe("browser-polyfill", () => {
+  describe("proxies non-wrapped functions", () => {
+    it("should proxy getters and setters", () => {
+      const fakeChrome = {
+        runtime: {myprop: "previous-value"},
+        nowrapns: {nowrapkey: "previous-value"},
+      };
+      return setupTestDOMWindow(fakeChrome).then(window => {
+        const setResult = window.browser.runtime.myprop = "new-value";
+        const setResult2 = window.browser.nowrapns.nowrapkey = "new-value";
+
+        assert.equal(setResult, "new-value",
+                     "Got the expected result from setting a wrapped property name");
+        assert.equal(setResult2, "new-value",
+                     "Got the expected result from setting a wrapped property name");
```

Comment:
We should check that these are also present in the `fakeChrome` object. And set multiple properties, with different values.

Also, we need to test two separate cases: 1) when we set the property before we ever read it, and 2) when we read the property, set it, and then read it again.